### PR TITLE
[video_player_avplay]No return empty error message

### PR DESCRIPTION
**Pull request by xiaowei-guan**
Fix https://github.com/flutter-tizen/plugins/issues/701

### CODE REVIEW

File: packages/video_player_avplay/tizen/src/plus_player.cc [hunk 1]
```diff
@@ -657,20 +657,106 @@ void PlusPlayer::OnResourceConflicted(void *user_data) {
   self->SendError("PlusPlayer error", "Resource conflicted");
 }
 
+std::string GetErrorMessage(plusplayer::ErrorType error_code) {
+  switch (error_code) {
+    case plusplayer::ErrorType::kNone: {
+      return "Successful";
+    }
```
Comment (swift-kim) [inline packages/video_player_avplay/tizen/src/plus_player.cc:664]: The braces seem redundant.

**Review by swift-kim — APPROVED** [review]
Otherwise looks good to me.

You could also increase the package version.

File: packages/video_player_avplay/tizen/src/plus_player.cc (hunk 1 above)
Comment (xiaowei-guan) [reply packages/video_player_avplay/tizen/src/plus_player.cc]: Done

**Comment by xiaowei-guan** [comment]
> Otherwise looks good to me.
> 
> You could also increase the package version.

Done

**Pull request merged**